fe/fe-core/src/main/java/org/apache/doris/datasource/LoadScanProvider.java

```diff
@@ -95,10 +94,7 @@ public FileLoadScanNode.ParamCreateContext createContext(Analyzer analyzer) thro
         if (fileGroupInfo.getSequenceMapCol() != null) {
             params.setSequenceMapCol(fileGroupInfo.getSequenceMapCol());
         }
-        if (fileFormatProperties.getFormatName().equals("hive_text")) {
-            params.setTextSerdeType(TTextSerdeType.HIVE_TEXT_SERDE);
-        }
-        params.setProperties(fileGroupInfo.getBrokerDesc().getBackendConfigProperties());
+        params.setProperties(fileGroupInfo.getBrokerDesc().getProperties());
```

Comment:
looks like createContext should not be called in latest master, could you please show an example that how this function is called?